go/cmd/vttablet/cli/cli.go

```diff
@@ -143,7 +143,6 @@ func run(cmd *cobra.Command, args []string) error {
 
 	qsc, err := createTabletServer(ctx, env, config, ts, tabletAlias, srvTopoCounts)
 	if err != nil {
-		ts.Close()
```

Comment:
This is required to fix the panic seen in shutdown - 
```
goroutine 1 [running]:
vitess.io/vitess/go/vt/topo.(*Server).Close(0xc0002e63c0)
	vitess.io/vitess/go/vt/topo/server.go:344 +0x38
vitess.io/vitess/go/cmd/vttablet/cli.run(0x45df020, {0xc00055e000?, 0x7?, 0x2414f70?})
	vitess.io/vitess/go/cmd/vttablet/cli/cli.go:176 +0x8b0
github.com/spf13/cobra.(*Command).execute(0x45df020, {0xc0001c6010, 0x1e, 0x1f})
	github.com/spf13/cobra@v1.8.0/command.go:983 +0xaca
github.com/spf13/cobra.(*Command).ExecuteC(0x45df020)
	github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(0x0?)
	github.com/spf13/cobra@v1.8.0/command.go:1039 +0x13
main.main()
	vitess.io/vitess/go/cmd/vttablet/vttablet.go:26 +0x1e
```

Since we have `defer ts.Close()` already above. Trying to close the topo server twice leads to panic.

Reply:
this will be fixed by https://github.com/vitessio/vitess/pull/17094